MHacks/RoundedView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RoundedView.swift
+//  MHacks
+//
+//  Created by Connor Krupp on 9/11/16.
+//  Copyright © 2016 MHacks. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class RoundedView: UIView {
```

Comment:
I'm pretty sure you can set this in the storyboard too, using User Defined Runtime Attributes